controllers/rollingupgrade_controller_test.go

```diff
@@ -1138,28 +1138,6 @@ func TestRunRestackSameLaunchConfig(t *testing.T) {
 	g.Expect(err).To(gomega.BeNil())
 }
 
-func TestRunRestackRollingUpgradeNotInMap(t *testing.T) {
```

Comment:
Any particular reason we are removing this test?

Reply:
This test assumes that runRestack won't be adding this rollingUpgrade object to the sync map.
But by calling the populateAsg(ruObj) we will also be adding the rollingUpgrade object to the sync map. populateAsg() calls Store function. Hence the purpose of this test is defeated.

Reply:
<img width="774" alt="Screen Shot 2021-04-13 at 9 31 52 AM" src="https://user-images.githubusercontent.com/7680410/114588065-1dafdc00-9c3b-11eb-84e0-7c387b215f1a.png">